nip47/publish_nip47_info.go

```diff
@@ -11,7 +12,7 @@ import (
 	"github.com/nbd-wtf/go-nostr"
 )
 
-func (svc *nip47Service) PublishNip47Info(ctx context.Context, relay nostrmodels.Relay, lnClient lnclient.LNClient) error {
+func (svc *nip47Service) PublishNip47Info(ctx context.Context, relay nostrmodels.Relay, appWalletPubKey string, appWalletPrivKey string, lnClient lnclient.LNClient) (*nostr.Event, error) {
```

Comment:
@frnandu I think for new apps the capabilities could be based on what permissions are set on the app (before we couldn't do this). I think @im-adithya mentioned this previously.

If we think this would be good to do, maybe this could be moved to a separate issue? we'd need to repost the event when the app permissions are updated. CC @bumi @reneaaron

Reply:
Yes, I can do that.

Reply:
I created a separate issue: https://github.com/getAlby/hub/issues/766, resolving this for now.